File: solutions/typescript/src/main7.ts
import * as fs from 'fs'

const inputs = fs.readFileSync('/dev/stdin', 'utf-8')
const a = inputs.split('\n')[1].split(' ').map(Number)

a.sort((l, r) => r - l)

let answer = a.reduce((sum, current, index) => {
  return sum + (index % 2 === 0 ? current : -current)
}, 0)
console.log(answer)

File: solutions/rust/main7.rs
use proconio::input;

fn main() {
    input! {
        n: u32,
        mut a: [u32; n],
    }

    a.sort();
    a.reverse();

    let mut answer = 0;
    for (index, value) in a.iter().enumerate() {
        if index % 2 == 0 {
            answer += value;
        } else {
            answer -= value;
        }
    }
    println!("{}", answer);
}
